### ISSUE TITLE
Suggestion: Combine all the "Product goes with X" values into one object.

### ISSUE THREAD

**Issue by Ernir** [davidarnarsson/vinbud-scraper#4]
As it is right now, about half the name/value pairs in each product object are dedicated to information about what kind of food items match the product.

This is somewhat cluttered. I believe a structure like this

```
{
    "goesWith": {
        "alifuglar": false,
        "eftirrettir": false,
        "fiskur": false,
        "graenmeti": true,
        "grill": false,
        "gris": false,
        "lamb": false,
        "lettariVillibrad": false,
        "naut": false,
        "ostur": false,
        "pasta": true,
        "villibrad": false,
        "skelfiskur": false
    },
    "description": "Ljósjarðarberjarautt. Léttkolsýrt, hálfsætt, létt fylling, fersk sýra. Jarðarber.",
    "id": "(21611)",
    "img": "http://www.vinbudin.is/ProductImages/ThumbnailSizeImages/21611.png",
    "link": "http://www.vinbudin.is/DesktopDefault.aspx/tabid-54?productID=21611",
    "price": "1.499 kr.",
    "reserve": false,
    "tilbuidAdDrekka": true,
    "title": "Black Tower Pink Bubbly ",
    "volume": "8,5%",
    "weight": "750 ml"
}
```

would be easier to navigate. Also, it's more legible, `product.goesWith.alifuglar` means something, `product.alifuglar` is kind of weird.

(An alternative restructuring would be to just list the names of the food categories in an array, but this way each item contains information about all the possibilities, which I think has some value.)

**Comment by davidarnarsson** [comment]
Sounds good to me, feel free to submit a PR :)

**Comment by davidarnarsson** [comment]
Whoops wrong button.

**Comment by davidarnarsson** [comment]
Fixed in v1.2.0
